app/models/kudo.rb

```diff
@@ -3,6 +3,7 @@
 class Kudo < ApplicationRecord
   belongs_to :employee, class_name: 'Employee'
   belongs_to :receiver, class_name: 'Employee'
+  belongs_to :company_value, class_name: 'CompanyValue'
```

Comment:
you can write it simply as follows - '_Convention Over Configuration_' is one of Rails' paradigm, which means basically that Rails is smart enough to guess some stuff (like name of the class here) if you are following the convention (which you are :)

```suggestion
  belongs_to :company_value
```